src/api_routes_v4.rs

```diff
@@ -277,6 +277,7 @@ pub fn config(cfg: &mut ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/like/list", get().to(list_comment_likes))
           .route("/save", put().to(save_comment))
           .route("/list", get().to(list_comments))
+          .route("/list/slim", get().to(list_comments_slim))
```

Comment:
I'm up for suggestions on what this should be also. It does need to be its own endpoint tho, because I want to guarantee its return type.